lib/matplotlib/widgets.py

```diff
@@ -324,6 +328,11 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt='%1.2f',
         if slidermax is not None and not hasattr(slidermax, 'val'):
             raise ValueError("Argument slidermax ({}) has no 'val'"
                              .format(type(slidermax)))
+        if orientation not in {'horizontal', 'vertical'}:
```

Comment:
While it's technically correct to use a set here, lists are the more common choice.